src/core/public/chrome/chrome_service.tsx

```diff
@@ -158,6 +161,29 @@ export class ChromeService {
         }
         this.collapsibleNavHeaderRender = render;
       },
+      registerNavLink: (useCase: ChromeUseCase, navLink: ChromeRegistrationNavLink) => {
```

Comment:
Since we're designing a public API, I feel the term `use case` is too specific because it's a product concept. Can we give it a more general name? something like `virtualDirectory`?

Reply:
Yeah, changed to `group`.